### PR TITLE
World map plotting

### PR DESCRIPTION
**Pull request by arebe337**

### CODE REVIEW

**Comment by arebe337** [comment]
@michaelweinold did all the adjustments and should be ready for review again

File: ADB/__pycache__/data_transformation_pandas.cpython-312.pyc
Comment (michaelweinold) [inline ADB/__pycache__/data_transformation_pandas.cpython-312.pyc:1]: `.pyc` files need to be added to the `.gitignore`.

File: ADB/Air_Traffic_Worldmap_year.pdf
Comment (michaelweinold) [inline ADB/Air_Traffic_Worldmap_year.pdf:1]: Cool!

**Review by michaelweinold — CHANGES_REQUESTED** [review]
...one file needs to be removed, otherwise this looks good.

**Comment by arebe337** [comment]
Corrected all comments @michaelweinold

**Pull request merged**